ci-operator/step-registry/baremetalds/assisted/operator/setup/baremetalds-assisted-operator-setup-commands.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds assisted operator setup command ************"
+
+# Fetch packet basic configuration
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/packet-conf.sh"
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF |& sed -e 's/.*auths\{0,1\}".*/*** PULL_SECRET ***/g'
+
+set -xeo pipefail
+
+function wait_for_crd() {
+  echo "Waiting for CRD (\$1) to be defined"
+  for i in {1..40}; do
+    oc get "crd/\$1" && break || sleep 10
+  done
+  oc wait --for condition=established --timeout=60s "crd/\$1" || exit 1
+}
+
+echo "Installing Hive..."
+cat <<EOCR | oc create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: hive-operator
+  namespace: openshift-operators
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: hive-operator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: hive-operator.v1.0.19
+EOCR
+
+wait_for_crd "clusterdeployments.hive.openshift.io"
+
+echo "Install PVCs and other prerequisites for assisted-installer operator..."
```

Comment:
I think that the PVC should be defined after the OLM prerequisites

Reply:
removed PVCs creation here
I'll fix the ``echo`` above as well

Reply:
We can probably do it in this order:
- Install operators
- Create local storage using LSO
- Create PVCs
- Run assisted-installer operand from assisted-installer operator

I think that should solve the order of operations for the 3 operators. That might be the current problem from what I'm seeing from the failures.

Reply:
no dependency problems, for the best of my knowledge